proxy-node-shared/src/test/java/uk/gov/ida/notification/shared/ProxyNodeLoggerTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.ida.notification.shared;
+
+import org.junit.Test;
+
+import java.util.logging.Level;
+
+import static org.mockito.AdditionalMatchers.not;
+import static org.mockito.ArgumentMatchers.contains;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doCallRealMethod;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class ProxyNodeLoggerTest {
+    private static final String LOG_MESSAGE = "stop your messing around";
+
+    @Test
+    public void shouldNotLogProxyNodeLoggerAsTheLogLocation() {
+        ProxyNodeLogger mockLogger = mock(ProxyNodeLogger.class);
+        doCallRealMethod().when(mockLogger).log(Level.INFO, LOG_MESSAGE);
```

Comment:
If we're going to call the real method we might as well use the real object ;)
`ProxyNodeLogger` can be initialised easily without any args, and we can use Mockito's `@Spy` annotation to verify method calls. We will also need to use the Mockito JUnit runner.